cocos2d/webview/webview-impl.js

```diff
@@ -122,7 +122,6 @@ let WebViewImpl = cc.Class({
         if (WebViewImpl._polyfill.enableBG)
             this._div.style["background"] = "#FFF";
 
-        this._div.style["background"] = "#FFF";
```

Comment:
The transparency of the iframe setting its child node is invalid because of the background color setting on dev